extensions/admin-tool-ext-win/package.json

```diff
@@ -70,6 +70,6 @@
 		"vscode-nls": "^3.2.1"
 	},
 	"devDependencies": {
-		"vscode": "1.0.1"
+		"vscode": "^1.1.33"
```

Comment:
do you mean 1.33.1?

Reply:
No, the npm package version is different than vscode version. The latest npm package is 1.1.33.